firebase/functions/src/shoppingListCleaner.ts

```diff
@@ -0,0 +1,21 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+
+export const ShoppingListCleaner = functions.pubsub.schedule("0 5 1-31/2 * *") // Every second (ODD) day @  05:00
+   .timeZone("Europe/Copenhagen")
+   .onRun(() => {
+      // Retrive all the entries that should be kept:
+      return admin.database().ref("vimangler")
+         .orderByChild("checked")
+         .equalTo(null)
```

Comment:
Værdierne kan både være null og false. 
Hvis man checker og derefter unchecker, så vil værdien være false.

Reply:
Meh. Kunne man lave `!true` mon?

Reply:
Nej ikke med equalTo. Der findes ikke en notEqualTo, så tror næsten den anden løsning du beskriver er bedre.

Reply:
Det vil resultere i en equalTo(false) og den fanger ikke null :(

Reply:
Ah well, troede egentligt at `false == null` i Typescript og `===` var det korrekte tjek for det specifikke output.